scripts/build/packaging.ts

```diff
@@ -0,0 +1,49 @@
+import {existsSync, readdirSync, unlinkSync} from "fs"
+import {BundleType as BundleType} from "./bundles"
+
+import {
+  asyncCopyTo,
+} from "./utils"
+
+export function getBundleOutputPaths(bundleType: BundleType, filename: string, packageName: string) {
+  const _filename = filename.replace(/^@lingui\//, "")
```

Comment:
Isn't this replace now outdated as you have removed the `@lingui` prefix from all package names in the `bundles.ts` file?